.github/workflows/ci_action.yml

```diff
@@ -35,6 +35,33 @@ jobs:
       - name: Check code compliance with pre-commit validators
         run: pre-commit run --all-files
 
+  check-code-pylint-and-mypy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ env.CHECKOUT_BRANCH }}
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.8"
```

Comment:
Should not be 3.7 oldest that is still supported ?

Reply:
i'd use the same one as for commit hooks